scripts/centos-req.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+sudo yum groupinstall -y "Development tools"
+sudo yum install -y gmp-devel mpfr-devel libmpc-devel zlib-devel vim git java java-devel
+curl https://bintray.com/sbt/rpm/rpm | sudo tee /etc/yum.repos.d/bintray-sbt-rpm.repo
+sudo yum install -y sbt texinfo gengetopt
+sudo yum install -y expat-devel libusb1-devel ncurses-devel cmake "perl(ExtUtils::MakeMaker)"
+# deps for poky
+sudo yum install -y python36 patch diffstat texi2html texinfo subversion chrpath git wget
+# deps for qemu
+sudo yum install -y gtk3-devel
+# deps for firemarshal
+sudo yum install -y python36-pip python36-devel rsync libguestfs-tools makeinfo yac bison flex expat glib-devel ctags
```

Comment:
Not your fault, but `yac` should be `yacc` - this is a typo in `centos-requirements.txt` that I just noticed.
```suggestion
sudo yum install -y python36-pip python36-devel rsync libguestfs-tools makeinfo expat ctags
```
Some packages (yacc, bison, flex) are already installed with `Development tools`.